bluetail/helpers.py

```diff
@@ -110,3 +116,75 @@ def get_tenderer_context(self, tenderer):
         tenderer_context["total_errors"] = len(errors)
 
         return tenderer_context
+
+
+class UpsertDataHelpers:
+
+    def upsert_ocds_data(self, ocds_json_path_or_string):
+        """
+        Takes a path to an OCDS Package or a stringn containing OCDS JSON data
+        Upserts all releases to the Bluetail database
+        """
+        if os.path.exists(ocds_json_path_or_string):
+            ocds_json = json.load(open(ocds_json_path_or_string))
+        else:
+            ocds_json = json.loads(ocds_json_path_or_string)
+
+        ocds_releases = []
+
+        if ocds_json.get("records"):
+            # We have a record package
+            for record in ocds_json["records"]:
+                compiledRelease = ocds_json["compiledRelease"]
+                ocds_releases.append(compiledRelease)
```

Comment:
Well spotted! This `records` loop isn't currently used as our data so far is all in `release packages` 
https://standard.open-contracting.org/latest/en/schema/release_package/ 
I added this quickly as a pre-emptive handling of different `record packages` in case we throw them at it later. 
https://standard.open-contracting.org/latest/en/schema/record_package/
I'll push the fix